src/main/java/org/testng/IObjectFactory2.java

```diff
@@ -5,12 +5,11 @@
  * Factory used to create all test instances. This object factory only receives the class
  * in parameter.
  *
- * @see org.testng.IObjectFactory
- *
  * @author Cedric Beust <cedric@beust.com>
- *
+ * @see org.testng.IObjectFactory
  * @since 5.14.6
  */
 public interface IObjectFactory2 extends ITestObjectFactory {
```

Comment:
Please have this interface contents moved to its parent and deprecate this interface.